app/views/users/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<center>
+<h1>Welcome to Viewing Party</h1>
+
+<p>Please Register for an Account</p>
+
+<div id="registration">
+  <%= form_with url: "/registration", local: true do |form| %>
```

Comment:
Would be a good idea to refactor all paths to using routes pathing nomenclature.